samples/snippets/src/main/java/com/example/spanner/TransactionWithTagSample.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner;
+
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.Options;
+import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.Statement;
+import com.google.cloud.spanner.TransactionContext;
+import com.google.cloud.spanner.TransactionRunner.TransactionCallable;
+
+/**
+ * Sample showing how to add and query tags to Cloud Spanner operations.
+ */
+public class TransactionWithTagSample {
+
+  // [START spanner_set_transaction_and_request_tags]
+  static void taggedTransaction(DatabaseClient databaseClient) {
+    // The request below would have transaction_tag set as "app=spanner,env=test".
+    databaseClient
+        .readWriteTransaction(Options.tag("app=spanner,env=test"))
+        .run(
+            new TransactionCallable<Void>() {
+              @Override
+              public Void run(TransactionContext transaction) throws Exception {
+                // The request below would have request_tag set as
+                // "app=spanner,env=test,action=select".
+                ResultSet queryResultSet = transaction
+                    .executeQuery(
+                        Statement.of("SELECT SingerId, FirstName, LastName FROM " + table),
+                        Options.tag("app=spanner,env=test,action=select"));
+                while (queryResultSet.next()) {
+                  System.out.printf(
+                      "%d %s %s\n",
+                      queryResultSet.getLong(0), queryResultSet.getString(1),
+                      queryResultSet.getString(2));
+                }
+
+                transaction
+                    .buffer(Mutation.newInsertOrUpdateBuilder("Singers")
+                        .set("SingerId").to(100)
+                        .set("FirstName").to("George")
+                        .set("LastName").to("Washington")
+                        .build());
+                return null;
+              }
+            });
+  }
+  // [END spanner_set_transaction_and_request_tags]
+
+  // [START spanner_query_tags]
+  static void queryStats(DatabaseClient databaseClient) {
+    // Execute query on Query statistics
+    // see: https://cloud.google.com/spanner/docs/introspection/query-statistics, for more details.
+    String sql =
+        "SELECT t.REQUEST_TAG, t.AVG_LATENCY_SECONDS, t.AVG_CPU_SECONDS "
```

Comment:
As far as I can tell, `REQUEST_TAG` is not yet available in (public) Cloud Spanner databases. This should therefore not be merged until the column is available.

Furthermore: The `t.` prefix here does not work. There's no alias `t` in the query below.